.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+name: rospkg-ci
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['*']
+
+jobs:
+    build:
+      strategy:
+        matrix:
+          os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, macos-latest]
+          python: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+          exclude:
+          - os: ubuntu-16.04
+            python: 3.6
+          - os: ubuntu-16.04
+            python: 3.7
+          - os: ubuntu-16.04
+            python: 3.8
+          - os: ubuntu-16.04
+            python: 3.9
+          - os: ubuntu-18.04
+            python: 3.5
+          - os: ubuntu-18.04
+            python: 3.6
+          - os: ubuntu-20.04
+            python: 2.7
+          - os: ubuntu-20.04
+            python: 3.5
+          - os: ubuntu-20.04
+            python: 3.6
+          - os: ubuntu-20.04
+            python: 3.7
+          - os: macos-latest
+            python: 3.5
+          - os: macos-latest
+            python: 3.6
+      name: rospkg tests
+      runs-on: ${{matrix.os}}
+
+      steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{matrix.python}}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{matrix.python}}
+      - name: Install dependencies
+        run: |
+          if test ${{matrix.python}} = 2.7; then
+            python -m pip install pyparsing==2.4.7
+          fi
+          python -m pip install --upgrade pip setuptools
+          python -m pip install nose coverage mock
+          python setup.py build develop
+      - name: Run tests
+        run: |
+          python -m nose --with-coverage --cover-package=rospkg --with-xunit test
+
+
```

Comment:
Not a huge fan of excess trailing whitespace...

Reply:
Thanks for spotting.